.github/workflows/release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         if: ${{ steps.read-version.outputs.version != '' }}
         id: check-release
         run: |
-          VERSION=$(git ls-remote --tags origin | grep ${{ steps.read-version.outputs.version }} || [[ $? == 1 ]] && echo '')
+          VERSION=$(git ls-remote --tags origin | grep --fixed-strings ${{ steps.read-version.outputs.version }} || [[ $? == 1 ]] && echo '')
```

Comment:
Grep needs to be instruct that this is a string, not a REGEX, otherwise `0.5.0` matches, for example, `0f5b0`.